prestoadmin/util/base_config.py

```diff
@@ -41,8 +79,8 @@ def prompt_user(self, conf):
                                 default=conf.get(self.key, self.default),
                                 validate=self.validate)
 
-    def collect_prompts(self, l):
-        l.append((self.prompt, self.key))
+    def get_keys(self):
```

Comment:
I don't remember why I had collect_prompts in here originally, but it was unused.